sdks/python/hatchet_sdk/clients/dispatcher/action_listener.py

```diff
@@ -275,15 +279,17 @@ async def _generator(self) -> AsyncGenerator[Action | None, None]:
 
                         break
 
-                    assigned_action, _, is_eof = t.result()
+                    result = t.result()
 
-                    if is_eof:
+                    if isinstance(result, UnexpectedEOF):
```

Comment:
Do we have a reliable repro/has this been tested?

Reply:
worked fine locally but happy to test against staging if you want!